### PR TITLE
Updated recipe for geoviews 1.5.0 release

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

**Comment by philippjfr** [comment]
@ocefpaf Thanks for adding this recipe originally. Would you mind adding me as a maintainer and merging this if it all looks good?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 package:
   name: geoviews
   version: {{ version }}
 
 source:
-  url: https://github.com/ioam/geoviews/archive/{{ version }}.tar.gz
-  sha256: eb4b9dbe5b8e2acd1b2a09141e72fc6cc5fef1116bf4e85cb014accb55a69eb2
+  fn: geoviews-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/geoviews/geoviews-1.5.0.zip
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We need the jinja var for the version so the bot can update it accordingly.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+{% set version = "1.5.0" %}
 
 package:
   name: geoviews
   version: {{ version }}
 
 source:
-  url: https://github.com/ioam/geoviews/archive/{{ version }}.tar.gz
-  sha256: eb4b9dbe5b8e2acd1b2a09141e72fc6cc5fef1116bf4e85cb014accb55a69eb2
+  fn: geoviews-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We don't need the `fn` entry, that info is in the `url`.

**Review by ocefpaf — APPROVED** [review]

**Comment by ocefpaf** [comment]
> Would you mind adding me as a maintainer

Yes please!

>  and merging this if it all looks good?

I made some minor changes. I'll merge as soon as we get green again.

Thanks!

**Comment by philippjfr** [comment]
Apologies, looks like I just clobbered your changes, hopefully they were identical anyway.

**Comment by ocefpaf** [comment]
> Apologies, looks like I just clobbered your changes, hopefully they were identical anyway.

No problem. Just updated the `CircleCI` token and I'll merge this as soon as it passes.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by philippjfr** [comment]
@ocefpaf Thank you!

**Comment by ocefpaf** [comment]
@philippjfr I commented out the line that adds the license for now but it would be nice to have it package in the next release.

**Comment by philippjfr** [comment]
Thanks, I'm not quite sure why that didn't get packaged but I'll raise the issue.

**Comment by ocefpaf** [comment]
> Thanks, I'm not quite sure why that didn't get packaged but I'll raise the issue.

Probably it is not listed in the `MANIFEST.in`. Python packaging recommendations/rules/best practices sucks when it comes to licenses strings and license files.

**Comment by philippjfr** [comment]
Okay thanks, just added it to the ``MANIFEST.in``. There should be a 1.5.1 soonish anyway.

**Comment by ocefpaf** [comment]
> Okay thanks, just added it to the ``MANIFEST.in``. There should be a 1.5.1 soonish anyway.

I'll merge this one to get you in as a maintainer so you'll be able to self-merge the 1.5.1 update :wink:

**Pull request merged**